tests/run_tests.sh

```diff
@@ -0,0 +1,42 @@
+#/usr/bin/env bash
+
+# This file is part of the MAVLink Router project
+#
+# Copyright (C) 2017  Intel Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO check if it's worth redirecting output to some file
+
+RESULT=true
+printf "\n--- Running unit tests ---\n"
```

Comment:
ugh. Let's not reinvent the wheel on running unit tests.

https://git.kernel.org/pub/scm/utils/kernel/kmod/kmod.git/tree/Makefile.am#n348
TESTS control all the tests to be run.

See "make check" in https://www.gnu.org/software/automake/manual/html_node/Standard-Targets.html

And: https://www.gnu.org/software/automake/manual/html_node/Tests.html

libell, bluez and systemd can also be used as a reference on how to integrate running testsuite